src/views/SystemProfiles/index.tsx

```diff
@@ -1,13 +1,143 @@
+    acronym: 'SYS3',
+    slug: 'system3',
+    link: '/system/system3'
+  },
+  {
+    id: '4',
+    name: 'System4',
+    acronym: 'SYS4',
+    slug: 'system4',
+    link: '/system/system4'
+  },
+  {
+    id: '5',
+    name: 'System5',
+    acronym: 'SYS5',
+    slug: 'system5',
+    link: '/system/system5'
+  }
+];
+
+const getStatusNotification = (status: string) => {
+  switch (status) {
+    case 'SUBMITTED':
+      return 'Status: The form has been submitted and is being reviewed.';
+    case 'REVIEWED':
+      return 'Status: Request Form has been reviewed. Prepare your business case.';
+    default:
+      return '';
+  }
+};
+
+type SystemProfilesProps = RouteComponentProps & {
+  auth: any;
+};
+
+export const SystemProfiles = ({ auth }: SystemProfilesProps) => {
+  const onSearch = () => {};
+  const getSuggestionValue = (suggestion: any): string => suggestion.name;
+  const renderSuggestion = (suggestion: any): string => suggestion.name;
+  const intakes = useSelector(
+    (state: AppState) => state.systemIntakes.systemIntakes
+  );
+  const searchResults = useSelector(
+    (state: AppState) => state.search.allSystemShorts
+  );
+  const [timeStamp, setTimeStamp] = useState('');
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const getSystemIntakes = async (): Promise<void> => {
+      dispatch(fetchSystemIntakes(await auth.getAccessToken()));
+      await setTimeStamp(
+        DateTime.local().toLocaleString(DateTime.DATETIME_MED)
+      );
+    };
+    getSystemIntakes();
+  }, [auth, dispatch]);
+
+  useEffect(() => {
+    const fetchSystemShorts = async (): Promise<void> => {
+      dispatch(getAllSystemShorts(await auth.getAccessToken()));
+    };
+    fetchSystemShorts();
+  }, [auth, dispatch]);
 
-const SystemProfiles = () => {
   return (
     <div>
-      <Header />
-      <h1>All System Profiles List Page</h1>
+      <Header>
+        <div className="system-profile__search-bar">
+          <SearchBar
+            name="System search"
+            onSearch={onSearch}
+            getSuggestionValue={getSuggestionValue}
+            renderSuggestion={renderSuggestion}
+            results={searchResults}
+          />
+        </div>
+
+        {mockSystems.length > 0 && (
+          <div className="system-profile__secondary-nav-wrapper">
+            <SecondaryNav
+              secondaryNavList={mockSystems.slice(0, 10)}
+              activeNavItem="all"
+            />
+          </div>
+        )}
+      </Header>
+      <div className="grid-container">
+        <UpcomingActions timestamp={timeStamp}>
+          {intakes &&
+            intakes.map(intake => {
+              return (
+                <Fragment key={intake.id}>
+                  <ActionBanner
+                    title={intake.projectName}
+                    helpfulText={getStatusNotification(intake.status)}
+                    label={
+                      <a href="/system/all">View submitted request form</a>
```

Comment:
The button/link in the `ActionBanner` will need to be configurable based on the action. For example, `View submitted request form` will be unstyled while something like `Begin Business Case` will be a CTA.

Instead of rendering a `<button>` in `ActionBanner`, we can use our `<Button />` in `src/components/shared/Button/index.tsx`. The default configuration is the blue button CTA. The `unstyled` prop should make it look like a link.